README.md

```diff
@@ -120,18 +123,27 @@ For example, if you run `npm dist-tags @types/react`, you'll see that TypeScript
 
 You may need to add manual [references](http://www.typescriptlang.org/docs/handbook/triple-slash-directives.html).
 
+</details>
+
 ## How can I contribute?
 
-Definitely Typed only works because of contributions by users like you!
+Definitely Typed only works because of contributions by users like you! 
 
 ### Testing
 
-Before you share your improvement with the world, use it yourself.
+Before you share your improvement with the world, use the types yourself by creating a `typename.d.ts` file in your project and filling out its exports:
+
+```ts
+declare module "libname" { 
+  // Types inside here
+  export function helloWorldMessage(): string
+}
+```
 
 #### Test editing an existing package
 
-To test local to your app, you can use [module augmentation](http://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation) to extend existing types from the DT module you want to work on.
-Alternatively, you can also edit the types directly in `node_modules/@types/foo/index.d.ts` to validate your changes, then bring the changes to this repo with the steps below.
+You can also edit the types directly in `node_modules/@types/foo/index.d.ts` to validate your changes, then bring the changes to this repo with the steps below.
+Alternatively, to quickly locally to your app, you can use [module augmentation](http://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation) to extend existing types from the DT module you want to work on.
```

Comment:
Missing a word between 'quickly' and 'locally'.

Reply:
You can also use it to override existing types, right?  If so, it could be useful to mention this as a way to debug types?

Reply:
Tested and you're right, `declare module` take precedence (is-array default is a function) 

![Screen Shot 2021-05-25 at 7 31 15 PM](https://user-images.githubusercontent.com/49038/119550043-ca7b8e00-bd8f-11eb-984c-5ba84b61c408.png)